Modules/Loadable/Annotations/MRML/vtkMRMLAnnotationAngleNode.cxx

```diff
@@ -381,7 +381,7 @@ int vtkMRMLAnnotationAngleNode::SetControlPoint(int id, double newControl[3])
 }
 
 //---------------------------------------------------------------------------
-double* vtkMRMLAnnotationAngleNode::GetPointColour()
+double* vtkMRMLAnnotationAngleNode::GetPointColor()
```

Comment:
Annotations will be gone, so it would be better to just leave the API as is (it would be frustrating for developers to update their code to according to the change and then see the classes completely removed soon after).

Reply:
What if I deprecate methods such as `GetPointColour` and call to `GetPointColor`?

If users are to be using Markups rather than Annotations, that deprecation should be indicated to them now.

Reply:
We should bug the users to move on to use Markups and not to spend any more time with Annotations.

Deprecation of Annotations module is planned in several steps - see the road map at https://github.com/Slicer/Slicer/wiki/Roadmap

Reply:
It appears that the "Deprecate Annotations module - April 2021" has been completed, but there hasn't been anything done in the code to indicate the deprecation?

Reply:
Correct. We have made the decision and told users and developers when the topic came up, but we haven't updated the documentation or added warnings in the code and user interface yet.

Reply:
Adding an entry to the migration guide would be sufficient: https://www.slicer.org/wiki/Documentation/Nightly/Developers/Tutorials/MigrationGuide#Slicer_5.0:_API_changes_since_4.10

Moving forward with the integration makes sense.

It may also be sensible to update the spellchecker to consider the words   `colour` and `behaviour` was "invalid"

Reply:
It does appear that codespell has a en_GB->en_US  dictionary so it may support setting a language to en_US. I'll investigate